packages/block-library/src/site-logo/index.php

```diff
@@ -72,4 +91,59 @@ function register_block_core_site_logo() {
 		)
 	);
 }
+
 add_action( 'init', 'register_block_core_site_logo' );
+
+/**
+ * Overrides the custom logo with a site logo, if the option is set.
+ *
+ * @param string $custom_logo The custom logo set by a theme.
+ *
+ * @return string The site logo if set.
+ */
+function _override_custom_logo_theme_mod( $custom_logo ) {
+	$site_logo = get_option( 'site_logo' );
+	return false === $site_logo ? $custom_logo : $site_logo;
+}
+
+add_filter( 'theme_mod_custom_logo', '_override_custom_logo_theme_mod' );
+
+/**
+ * Updates the site_logo option when the custom_logo theme-mod gets updated.
+ *
+ * @param string $custom_logo The custom logo set by a theme.
+ *
+ * @return string The custom logo.
+ */
+function _sync_site_logo_to_custom_logo( $custom_logo ) {
```

Comment:
I think this function name is reversed, as this is hooked into `pre_set_theme_mod_custom_logo`... should it be `_sync_custom_logo_to_site_logo` instead?

(and vise versa for the function below, as well)

Reply:
fixed in https://github.com/WordPress/gutenberg/pull/32229/commits/389aed8ad57f8ac1f5a9d01379f1408432101335 :+1: